doc/html/classes.html

```diff
@@ -4,7 +4,7 @@
 <head>
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
 <meta http-equiv="X-UA-Compatible" content="IE=9"/>
-<meta name="generator" content="Doxygen 1.10.0"/>
```

Comment:
Looks like `apt` installs an older doxygen then `brew` did on my Mac, not sure if it matters.